src/externs.d.ts

```diff
@@ -1,14 +1,7 @@
-import {Page as RealPage} from './Page.js';
 import * as child_process from 'child_process';
 
 declare global {
   module Puppeteer {
-    export class Page extends RealPage { }
-    /* TODO(jacktfranklin@): once DOMWorld, Page, and FrameManager are in TS
-     * we can remove this and instead use the type defined in LifeCycleWatcher
-     */
-    export type PuppeteerLifeCycleEvent = 'load' | 'domcontentloaded' | 'networkidle0' | 'networkidle2';
-
     export interface ConnectionTransport {
       send(string);
       close();
```

Comment:
in a follow up PR I will remove this file and find a new home for the interfaces we need where they are not exposed globally